docs/install.md

```diff
@@ -30,23 +30,26 @@ To install Compose, do the following:
 
 3. Go to the <a href="https://github.com/docker/compose/releases" target="_blank">Compose repository release page on GitHub</a>.
 
-4. Follow the instructions from the release page and run the `curl` command,
+4. Follow the instructions from the release page and run the `wget` or `curl` command below,
```

Comment:
we originally tried to avoid `wget` for simplicity and consistency - @thaJeztah can probably give a better over-view of where we're at atm.